public/app/plugins/datasource/cloudwatch/components/VariableQueryEditor/VariableQueryField.tsx

```diff
@@ -34,7 +34,7 @@ export const VariableQueryField = <T extends string | VariableQueryType>({
         onChange={({ value }) => onChange(value!)}
         options={options}
         isLoading={isLoading}
-        inputId="inline-field"
+        inputId={label}
```

Comment:
the `id` should be unique here (and also it seems that there is a property for this that's not being used.